here-naksha-app-service/src/test/java/com/here/naksha/app/auth/JwtTest.java

```diff
@@ -0,0 +1,101 @@
+package com.here.naksha.app.auth;
+
+
+import com.here.naksha.app.common.ApiTest;
+import com.here.naksha.app.common.NakshaTestWebClient;
+import com.here.naksha.app.common.TestUtil;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.net.http.HttpResponse;
+import java.util.UUID;
+
+import static com.here.naksha.app.common.CommonApiTestSetup.setupSpaceAndRelatedResources;
+import static com.here.naksha.app.common.TestUtil.generateJWT;
+import static com.here.naksha.app.common.TestUtil.loadFileOrFail;
+import static com.here.naksha.app.common.assertions.ResponseAssertions.assertThat;
+
+public class JwtTest extends ApiTest {
+    // For this test suite, the default test-config.json denotes that
+    // the service is launched in DUMMY auth mode
+
+    @BeforeAll
+    static void setup(){
+        setupSpaceAndRelatedResources(new NakshaTestWebClient(), "Auth/setup");
+    }
+
+    private static final String SPACE_ID = "auth_test_space";
+
+    @Test
+    public void testDummyModeNoJWT() throws Exception {
+        final String streamId = UUID.randomUUID().toString();
+        // Providing no JWT, the master token should be employed automatically
+        HttpResponse<String> response = getNakshaClient().get("hub/storages", streamId);
+        assertThat(response).hasStatus(200);
+    }
+
+    @Test
+    public void testDummyModeNoJWTAppIdAuthorExistsInXyzNamespace() throws Exception {
+        final String streamId = UUID.randomUUID().toString();
+        final String bodyJson = loadFileOrFail("Auth/NoJWTAppIdAuthorExists/create_features.json");
+        final String expectedBodyPart = loadFileOrFail("Auth/NoJWTAppIdAuthorExists/feature_response_part.json");
+
+        // Providing no JWT, the master token should be employed automatically
```

Comment:
In this test case, we should provide our own JWT with custom appId and userId values. And validate that xyz namespace gets populated with this custom values or not.
In that case, you would want to use different test folder name (instead of "NoJWTAppIdAuthorExists")